lib/wcc/contentful/simple_client.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require 'http'
+
+require_relative 'simple_client_response'
+
+module WCC::Contentful
+  class SimpleClient
+    def initialize(api_url:, space:, access_token:, **options)
+      @api_url = URI.join(api_url, '/spaces/', space + '/')
+      @space = space
+      @access_token = access_token
+
+      @get_http = options[:override_get_http] if options[:override_get_http].present?
+
+      @options = options
+      @query_defaults = {}
+      @query_defaults[:locale] = @options[:default_locale] if @options[:default_locale]
+    end
+
+    def get(path, query = {})
+      url = URI.join(@api_url, path)
+
+      Response.new(self,
+        { url: url, query: query },
+        get_http(url, query))
+    end
+
+    private
+
+    def get_http(url, query, headers = {}, proxy = {})
+      headers = {
+        Authorization: "Bearer #{@access_token}"
+      }.merge(headers || {})
+
+      q = @query_defaults.dup
+      q = q.merge(query) if query
+
+      resp =
+        if @get_http
+          @get_http.call(url, q, headers, proxy)
+        else
+          default_get_http(url, q, headers, proxy)
+        end
+      if [301, 302, 307].include?(resp.code) && !@options[:no_follow_redirects]
+        resp = get_http(resp.headers['location'], nil, headers, proxy)
+      end
+      resp
+    end
+
+    def default_get_http(url, query, headers = {}, proxy = {})
+      if proxy[:host]
+        HTTP[headers].via(proxy[:host], proxy[:port], proxy[:username], proxy[:password])
```

Comment:
decided to go with `HTTP`?

Reply:
That's just the default - you can inject Typhoeus on a per-app basis.  Copypasted straight out of the contentful.rb gem